sdk/typescript/src/types/common.ts

```diff
@@ -13,8 +13,9 @@ import {
 import { sha256Hash } from '../cryptography/hash';
 import { Ed25519PublicKey } from '../cryptography/ed25519-publickey';
 import { Secp256k1PublicKey } from '../cryptography/secp256k1-publickey';
+import { Base64DataBuffer } from '../serialization/base64';
 
-/** Base64 string representing the object digest */
+/** Base58 string representing the transaction digest */
```

Comment:
We should support both at the same time while this rolls out.

Reply:
good call. in the case of explorer gets deployed before sui, it should parse base64.